bokehjs/src/lib/models/filters/filter.ts

```diff
@@ -30,7 +30,7 @@ export class Filter extends Model {
 
   compute_indices(_source: DataSource): number[] | null {
     const filter = this.filter
-    if (filter != null && filter.length >= 0) {
+    if (filter != null && filter.length > 0) {
```

Comment:
The other clause should have been removed, otherwise users will get a misleading warning.

Reply:
Do you mean ```filter != null```?

Reply:
No, I mean the other clause. The final code should be:
```ts
if (filter != null) {
```

Reply:
@mattpap I've deleted the clause